fcs/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -24,7 +24,7 @@
     <Description>The F# compiler services package contains a custom build of the F# compiler that exposes additional functionality for implementing F# language bindings, additional tools based on the compiler or refactoring tools. The package also includes F# interactive service that can be used for embedding F# scripting into your applications.</Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <Authors>Microsoft Corporation; F# community contributors</Authors>
-    <PackageLicenseUrl>https://github.com/fsharp/FSharp.Compiler.Service/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Nice, thanks for this. I thought the NuGet server was going to eventually stop allowing packages to upload if they didn't specify this, but I guess not.